_src/assets/js/reset-button.js

```diff
@@ -1,96 +1,50 @@
-// import { userInfo } from "os";
-
-// El boton reset abre un pop-up para borrar o no la información añadida hasta el momento.
-
-//El botón YES resetea el formulario, esto es ,la paleta de colores en default (verdes), borra todos los inputs, la foto miniatura y la foto de preview. También tiene que dejar la tarjeta vacía y los iconos sin rodear.
-
-// Recoge el botón reset
+/* eslint-disable strict */
 const buttonResetEl = document.querySelector('.reset__btn');
-// Recoge el select por defecto(el primero)
 const selectEl = document.querySelector('#palette--blue-green');
-// Recoge todos los inputs del formulario
 const formEl = document.querySelectorAll('.fillTheGap-input');
-// Recoge la foto miniatura
-const miniatureEl = document.querySelector('.profile-miniature');
-// Recoge la foto del preview
-const photoPreviewEl = document.querySelector('.card__picture');
-// Recoge user name de la tarjeta
 const userNameEl = document.querySelector('.card-user__name');
-// Recoge user job de la tarjeta
 const userJobEl = document.querySelector('.card-user__position');
-//Button Yes
 const buttonResetYEl = document.querySelector('.btn-resetY');
-//Button NO
 const buttonResetNEl = document.querySelector('.btn-resetN');
-//Pop Up
 const containerAlertEl = document.querySelector('.alert');
-
-//Listeners de los botones Reset,No y Yes, respectivamente
+const alertReset = () => {
+  containerAlertEl.classList.add('alert-delete');
+};
 buttonResetEl.addEventListener('click', alertReset);
+const keepInfo = () => {
+  containerAlertEl.classList.remove('alert-delete');
+};
 buttonResetNEl.addEventListener('click', keepInfo);
+const resetInputs = () => {
+  for (let i = 0; i < formEl.length; i++) {
+    formEl[i].value = '';
+  }
+};
+const resetCard = () => {
+  userNameEl.innerHTML = 'User name';
+  userJobEl.innerHTML = 'User job';
+};
+const resetSN = () => {
+  const arrSN = document.querySelectorAll('.card__item-list');
+  for (let i = 0; i < arrSN.length; i++) {
+    arrSN[i].classList.remove('card__item-list');
+    arrSN[i].classList.add('card__item-list-no-border');
+  }
+};
+const resetColor = () => {
+  selectEl.checked = true;
```

Comment:
this function must add the default color

Reply:
Thanks Valeria, I just added it: "included add and remove of classlists in reset color function in partial reset-button.js"

Reply:
Great job!! <3